server/plugins/error-pages.plugin.js

```diff
@@ -17,6 +17,11 @@ module.exports = {
         if (response.isBoom) {
           const statusCode = response.output.statusCode
 
+          if (request.path === '/favicon.ico') {
```

Comment:
This should go above line 18 since there is no point in getting the status code if you are just going to return immediately after. Also, need to increment the build number